core/arch/arm/kernel/abort.c

```diff
@@ -546,6 +553,11 @@ static enum fault_type get_fault_type(struct abort_info *ai)
 	}
 }
 
+void __weak plat_abort_handler(struct abort_info *ai __unused)
+{
+	/* Do nothing */
+}
```

Comment:
Is this weak function needed?

Reply:
For whatever reason, if the switch is enabled and the function is not defined? Given that the switch is intended for this purpose, I think  there is indeed no need for this weak function.

Reply:
If the switch is enabled, the platform should implement the handler. Having a build error is a nice way to trigger that it's missing IMO.